lib/cisco_node_utils/command_reference_common_bgp.yaml

```diff
@@ -100,6 +100,11 @@ bgp:
     config_set_append: '<state> log-neighbor-changes'
     default_value: false
 
+  maxas_limit:
+    config_get_token_append: '/^maxas-limit (\d+)$/'
+    config_set_append: '<state> maxas-limit <limit>'
+    default_value: 50
```

Comment:
Where does 50 come from? Your comments indicated that there was no default value.